Cargo.toml

```diff
@@ -23,5 +23,5 @@ gl_generator = { version = "0.14", optional = true }
 walkdir = "2"
 
 [features]
-egl = ["gl_generator"]               # Only does anything on Windows
-build_dlls = ["egl", "gl_generator"]
```

Comment:
This is because because no build_dlls code directly depends on gl_generator and it gets included anyway because build_dlls depends on egl.